File: bionic-js-ios/BjsTests/Bjs/BjsTestProject.swift
import Foundation

@objc(BjsTestProject)
class BjsTestProject : BjsProject {
    
    override class func initialize(_ bjs: Bjs) {
        bjs.loadBundle(BjsTestProject.self, "test")
        bjs.addNativeWrapper(ToyComponent1BjsWrapper.self)
    }
}

File: bionic-js-runtime-java/src/test/java/acceptance/toy1/bjs/BjsTestProject.java
package acceptance.toy1.bjs;

import bionic.js.Bjs;
import bionic.js.BjsProject;
import bionic.js.BjsProjectTypeInfo;

public class BjsTestProject extends BjsProject
{
    @BjsProjectTypeInfo.Initializer
    public static void initialize(Bjs bjs)
    {
        initProject();
        bjs.loadBundle(BjsTestProject.class, "test");
        bjs.addNativeWrapper(ToyComponent1BjsExport.Wrapper.class);
        bjs.addNativeWrapper(ToyComponent2BjsExport.Wrapper.class);
    }
}
